### DISCUSSION TITLE
WUX::g_DirectUI_DXamlCore_GetPeerPrivate_Offset

### DISCUSSION THREAD

**Discussion by MuscularPuky** [valinet/ExplorerPatcher#3055] [General]
But the EP is working perfectly fine

![image](https://github.com/valinet/ExplorerPatcher/assets/93962018/868db2c9-45a2-4922-ba54-ee74606cc4d8)

build 26062~26080

**Comment by pyrates999** [comment]
Canary builds are not supported because the code is not final and can change with each new release.
